.lando.yml

```diff
@@ -53,35 +53,26 @@ services:
   #       - /dev/shm:/dev/shm
   #     command: /opt/bin/entry_point.sh
   # elasticsearch:
-  #   type: compose
-  #   meUser: elasticsearch
-  #   ssl: true
-  #   sslExpose: false
-  #   services:
-  #     image: blacktop/elasticsearch:7
-  #     command: /elastic-entrypoint.sh elasticsearch
-  #     ports:
-  #       - "9200:9200"
-  #       - "9300:9300"
-  #     volumes:
-  #       - ./.lando/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml
+  #   type: elasticsearch:7
+  #   portforward: 9200
+  #   mem: 1050m
   #   run_as_root:
-  #     # Install `analysis-icu` plugin after ES service boots up
-  #     - elasticsearch-plugin install analysis-icu
-  #   overrides:
-  #     environment:
-  #       JAVA_HOME: /usr/lib/jvm/default-jvm
-  #       ES_JAVA_OPTS: -Xms512m -Xmx512m
+  #     - |
+  #       /bin/sh -c "
+  #       if [ ! -d '/opt/bitnami/elasticsearch/plugins/analysis-icu' ]; then
+  #         elasticsearch-plugin install analysis-icu
+  #       fi
+  #       "
   # kibana:
   #   type: compose
   #   meUser: kibana
   #   services:
-  #     image: blacktop/kibana:7
-  #     command: /docker-entrypoint.sh kibana
+  #     image: bitnami/kibana:7.7.0
```

Comment:
We usually try to avoid that specific version numbers for template maintainability reasons. Also, this should match `elasticearch` version number.